src/common/config_opts.h

```diff
@@ -166,6 +166,13 @@ OPTION(heartbeat_file, OPT_STR, "")
 OPTION(heartbeat_inject_failure, OPT_INT, 0)    // force an unhealthy heartbeat for N seconds
 OPTION(perf, OPT_BOOL, true)       // enable internal perf counters
 
+OPTION(torrent_tracker, OPT_STR, "")    // torrent_tracker
+OPTION(torrent_createby, OPT_STR, "")    // torrent_createby
+OPTION(torrent_comment, OPT_STR, "")    // torrent_comment
+OPTION(torrent_encoding, OPT_STR, "")    // torrent_encoding
+OPTION(torrent_origin, OPT_STR, "")    // torrent_origin
+OPTION(sha_unit, OPT_INT, 512*1024)    //sha_unit 521K
```

Comment:
And this one should be rgw_torrent_sha_unit (sha_unit itself it too generic a name)